packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -277,6 +277,9 @@ const onShowSettingsDialog = (target: AvailableSettingsMenuKeys) => {
 onResult((queryResult) => {
   if (queryResult.data?.workspaceBySlug) {
     workspaceMixpanelUpdateGroup(queryResult.data.workspaceBySlug)
+    useHead({
```

Comment:
Not sure if we sanitize user data, but might be worth to check `useHeadSafe` because workspace name is user input based; https://nuxt.com/docs/api/composables/use-head-safe